src/views-components/context-menu/index.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) The Arvados Authors. All rights reserved.
+//
+// SPDX-License-Identifier: AGPL-3.0
+
+import { ContextMenuHOC, addMenuActionSet } from "./context-menu";
+import { projectActionSet } from "./action-sets/project-action-set";
+import { rootProjectActionSet } from "./action-sets/root-project-action-set";
+
+export default ContextMenuHOC;
+
+export enum ContextMenuKind {
+    RootProject = "RootProject",
```

Comment:
ROOT_PROJECT

Reply:
The majority of enums uses SnakeCase right now. Not sure if we should treat them as constants and use CONSTANT_NAMING or as a data type and use SnakeCase.